apps/captain_fact/lib/captain_fact/videos/videos.ex

```diff
@@ -153,4 +155,22 @@ defmodule CaptainFact.Videos do
 
     Enum.group_by(Repo.all(query), &elem(&1, 0), &elem(&1, 1))
   end
+
+  @doc """
+  Download and store captions for a video.
+  Returns captions if success or {:error, reason} if something bad happend.
+
+  Usage:
+  iex> download_captions(video, "en")
+  iex> download_captions(video, "fr")
+  """
+  def download_captions(video = %Video{}, locale) do
```

Comment:
The `locale` is now passed in. This could also be removed as the video already has a `language`.

Reply:
Same here, my first thought was that we could have multiple subtitles for a video. But regarding the lack of information with this YouTube endpoint your choice of having only one caption track per video is good and we don't need to store a locale for it.